core_v2/sources/domains.move

```diff
@@ -53,6 +55,16 @@ module aptos_names_v2::domains {
     const EVALID_SIGNATURE_REQUIRED: u64 = 16;
     /// The domain is too short.
     const EDOMAIN_TOO_SHORT: u64 = 17;
+    /// The name is not expired in 6 months, thus not eligible for renewal
+    const EDOMAIN_NOT_AVAILABLE_TO_RENEW: u64 = 18;
+    /// The subdomain is not eligible for renewal
+    const ESUBDOMAIN_IS_AUTO_RENEW: u64 = 19;
+    /// The name is expired
+    const ENAME_EXPIRED: u64 = 20;
+    /// The subdomain not exist
+    const ESUBDOMAIN_NOT_EXIST: u64 = 21;
+    /// The subdomain not exist
+    const ENOT_SUBDOMAIN: u64 = 22;
```

Comment:
what's the difference between `ESUBDOMAIN_NOT_EXIST` and `ENOT_SUBDOMAIN`?

Reply:
The name is not a subdomain. sorry forgot to change the comment :P